spec/features/orders/user_can_view_a_single_order_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe "when they visit the show page for an order" do
+  it "shows item title, price, quantity" do
+    user = Fabricate(:user)
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+    category = Category.create(name: "Joke Products")
```

Comment:
I think that our item fabricator automatically fabricates a category with each item. Does this test require all items to be the same category? If not we may be able to refactor this a bit.